src/modules/cmdpal/Microsoft.CmdPal.Core.ViewModels/Commands/ShowDetailsCommand.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using CommunityToolkit.Mvvm.Messaging;
+using Microsoft.CmdPal.Core.ViewModels.Messages;
+using Microsoft.CommandPalette.Extensions.Toolkit;
+
+namespace Microsoft.CmdPal.Core.ViewModels.Commands;
+
+public sealed partial class ShowDetailsCommand : InvokableCommand
+{
+    public static string ShowDetailsCommandId { get; } = "com.microsoft.cmdpal.showDetails";
+
+    private static IconInfo IconInfo { get; } = new IconInfo("\uF000"); // KnowledgeArticle Icon
+
+    private DetailsViewModel Details { get; set; }
+
+    public ShowDetailsCommand(DetailsViewModel details)
+    {
+        Id = ShowDetailsCommandId;
+        Name = "Show Details";
```

Comment:
```suggestion
        Name = "Show details";
```

As per Windows writing style guidelines, we should use Sentence casing.

Also, do we need to loc this?